spack/package.py

```diff
@@ -86,7 +86,7 @@ def url_for_version(self, version):
     )
     variant(
         "ipolates",
-        default="3",
+        default="0",
         description="Use to interpolate to new grids (0 = OFF, 1 = ip, 3 = ip2)",
         values=("0", "1", "3"),
```

Comment:
This is surely not correct, there's no longer an ip2...

Reply:
Should I just delete all references to ipolates = 3 and change it so we're only checking a boolean value? If so, would this cause issues if we decided we wanted to have checks where we build older versions of wgrib2?

Reply:
We will not be testing any older versions of wgrib2. Is there any other reason not to change it?

Reply:
Nope. I'll go ahead and change it.